specification/metadata/schema/providerversion.md

```diff
@@ -0,0 +1,22 @@
+# Provider Version
+
+The ProviderVersion MAY be provided to allow the provider to declare the version of logic by which the FOCUS dataset was generated. ProviderVersion allows for the provider to specify changes that may not result in a structural change in the data.
```

Comment:
Is it worth being explicit that the provider version is separate from the FOCUS version? I think this is important to prevent confusion and I could anticipate this happening.

Reply:
We could explain a single FOCUS version could be referenced for multiple provider versions.